tests/e2e/test_sampling_params.py

```diff
@@ -0,0 +1,275 @@
+import pytest
+from vllm import LLM, SamplingParams
+
+pytestmark = pytest.mark.basic
+
+
+@pytest.fixture(scope="module")
+def spyre_model() -> LLM:
+    return LLM(model="ibm-granite/granite-3.1-2b-instruct", max_model_len=128)
+
+
+def test_spyre_batch1_temperature(spyre_model: LLM, backend, monkeypatch,
+                                  use_llm_cache):
+    monkeypatch.setenv("VLLM_SPYRE_DYNAMO_BACKEND", backend)
```

Comment:
Oh I actually liked the earlier way in which we had a fixture to set this `monkeypatch`. We can always use the `backend` feature in that fixture and avoid duplication of this line across all tests